apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -97,6 +102,55 @@ export class ReminderEmailsService {
     return []
   }
 
+  async getUnmatchedMenteesFor45Days() {
+    const approvedDate = new Date()
+    approvedDate.setDate(approvedDate.getDate() - 45)
+
+    // 1st Step: Get all mentees that have been approved 45 days ago
+    const approvedMenteesFrom45DaysAgo = await this.conProfilesService.findAll({
+      'RecordType.DeveloperName': UserType.MENTEE,
+      Profile_Status__c: ConnectProfileStatus.APPROVED,
+      Profile_First_Approved_At__c: {
+        $eq: jsforce.SfDate.toDateLiteral(approvedDate),
+      },
+    })
+
+    const approvedMenteeIds = approvedMenteesFrom45DaysAgo.map(
+      (mentee) => mentee.props.userId
+    )
+
+    if (approvedMenteeIds.length === 0) return []
+
+    // 2nd Step: Get all mentorship matches where the mentee is one of the approved mentees above
+    const mentorshipMatches = await this.conMentorshipMatchesService.findAll({
```

Comment:
The comment confuses me. Are we getting ALL mentorship matches... or only mentorship matches at the `APPLIED` stage?